eclair-core/src/main/scala/fr/acinq/eclair/channel/InteractiveTxBuilder.scala

```diff
@@ -763,10 +763,10 @@ private class InteractiveTxBuilder(replyTo: ActorRef[InteractiveTxBuilder.Respon
   }
 
   def signFundingTx(completeTx: SharedTransaction, commitments: Commitments): Behavior[Command] = {
-    val shouldSignFirst = if (fundingParams.localAmount < fundingParams.remoteAmount) {
+    val shouldSignFirst = if (completeTx.localAmountIn < completeTx.remoteAmountIn) {
       // The peer with the lowest total of input amount must transmit its `tx_signatures` first.
       true
-    } else if (fundingParams.localAmount == fundingParams.remoteAmount) {
+    } else if (completeTx.localAmountIn == completeTx.remoteAmountIn) {
       // When both peers contribute the same amount, the peer with the lowest pubkey must transmit its `tx_signatures` first.
       LexicographicalOrdering.isLessThan(commitments.localParams.nodeId.value, commitments.remoteNodeId.value)
     } else {
```

Comment:
Nit:

```scala
    val shouldSignFirst = if (completeTx.localAmountIn == completeTx.remoteAmountIn) {
      // When both peers contribute the same amount, the peer with the lowest pubkey must transmit its `tx_signatures` first.
      LexicographicalOrdering.isLessThan(commitments.localParams.nodeId.value, commitments.remoteNodeId.value)
    } else {
      // The peer with the lowest total of input amount must transmit its `tx_signatures` first.
      completeTx.localAmountIn < completeTx.remoteAmountIn
    }
```

Reply:
Thanks, done in https://github.com/ACINQ/eclair/pull/2501/commits/2fa206ca0b9b0185081d98e57d95f085eb03f31a